certdocs/TRAQ-138-SDD.md

```diff
@@ -93,55 +93,55 @@ Reqtraq will check the validity of a certification document name by checking the
+<a name="#REQ-TRAQ-SWL-1">REQ-TRAQ-SWL-1</a>
 ```
 
-- each requirement referred to in the document is replaced with a link to that requirement. For example a reference to the High Level Requirement REQ-TRAQ-SYS-001 will be replaced with:
+- each requirement referred to in the document is replaced with a link to that requirement. For example a reference to the High Level Requirement REQ-TRAQ-SYS-1 will be replaced with:
 
 ```
-<a href="https://a/REQ-TRAQ-SYS-001">REQ-TRAQ-SYS-001</a>
+<a href="https://a/REQ-TRAQ-SYS-1">REQ-TRAQ-SYS-1</a>
 ```
 
 Where `http://a` is the Daedalean URL redirector. The URL redirector will infer the name of the document where the requirement is defined (TRAQ-137-SRD in our case) and use the Google Drive API or other methods to find the URL of the document, then redirect to it.
 
-Note that the URL redirector needs to infer the name of the document the requirement is defined in, as described in REQ-TRAQ-SWL-002.
+Note that the URL redirector needs to infer the name of the document the requirement is defined in, as described in REQ-TRAQ-SWL-2.
 
 ###### Attributes:
 - Rationale: this ensures that all documents defining or reference requirements don’t introduce typing mistakes.
-- Parents: REQ-TRAQ-SWH-012
+- Parents: REQ-TRAQ-SWH-12
 - Verification: Unit test
 - Safety impact: None
 
 
-##### REQ-TRAQ-SWL-002 Construct the requirement URL.
+##### REQ-TRAQ-SWL-2 Construct the requirement URL.
 
-The <abbr title="Requirements Management Tool">RMT</abbr> SHALL infer the document where a requirement is defined solely based on the name of the requirement. The correct name as defined by REQ-TRAQ-SWL-020 can be uniquely constructed as follows:
-- take the second part of the requirement id, representing the project/system abbrev, e.g. "TRAQ" for "REQ-TRAQ-SYS-001"
+The <abbr title="Requirements Management Tool">RMT</abbr> SHALL infer the document where a requirement is defined solely based on the name of the requirement. The correct name as defined by REQ-TRAQ-SWL-20 can be uniquely constructed as follows:
+- take the second part of the requirement id, representing the project/system abbrev, e.g. "TRAQ" for "REQ-TRAQ-SYS-1"
 - depending on the third part of the requirement id, representing the requirement type, append the document type:
   - for SWL and <abbr title="Hardware Low-Level Requirement">HWL</abbr>, append `-138-SDD`
   - for SWH and <abbr title="Hardware High-Level Requirement">HWH</abbr>, append `-137-SRD`
   - for <abbr title="System-Level Requirement">SYS</abbr>, append `-100-ORD`
 
 ###### Attributes:
 - Rationale: simplicity and completeness: the ability to find a requirement only based on its name simplifies the development and the verification process.
-- Parents: REQ-TRAQ-SWH-001
+- Parents: REQ-TRAQ-SWH-1
 - Verification: Unit test
 - Safety impact: None
 
 
-##### REQ-TRAQ-SWL-003 Uniform requirement ID format.
+##### REQ-TRAQ-SWL-3 Uniform requirement ID format.
```

Comment:
updated

another question related to github workflows: I see in red `Changes requested`, and also `See review` and `Dismiss review`, even though I pushed a change. What am I supposed to do to resolve this?